application/config/module.config.php

```diff
@@ -247,4 +247,44 @@
             'image/gif'  => 'image',
         ),
     ),
+    'oembed' => array(
+        'whitelist' => array(
+            '#http://(www\.)?youtube\.com/watch.*#i',
```

Comment:
Seems like these first 6 youtube ones should just be condensed into 3 with the `https?://` construction